var/spack/repos/builtin/packages/adol-c/package.py

```diff
@@ -44,8 +44,7 @@ class AdolC(AutotoolsPackage):
     variant('doc',      default=True,  description='Install documentation')
     variant('openmp',   default=False, description='Enable OpenMP support')
     variant('sparse',   default=False, description='Enable sparse drivers')
-    variant('tests',    default=True,
-            description='Build all included examples as a test case')
+    variant('examples', default=True,  description='Install examples')
```

Comment:
Kept this one as a variant because it actually controls what gets installed and therefore should change the hash.